.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseSetParser.cs

```diff
@@ -32,6 +32,8 @@ public DateTimeParseResult Parse(ExtractResult er, DateObject refDate)
 
             if (er.Type.Equals(ParserName, StringComparison.Ordinal))
             {
+                er.Text = er.Text.Trim();
```

Comment:
Why do we need this trim now?

Reply:
missed to removed it earlier. corrected it now.